### PR TITLE
New overflow delete

### PR DESCRIPTION
**Pull request by rdavidwu**
Added new overflow-menu fixed button
Added fixed delete button to actionbar_sample

### CODE REVIEW

**Comment by jasondscott** [comment]
Can you add the approipreate documentation on how to use this feature to docs/README.md ?

**Comment by rdavidwu** [comment]
Added the documentation right under "action overflow"

File: src/lib/panel.js [hunk 1]
```diff
@@ -320,6 +345,14 @@ $.widget( "mobile.panel", $.mobile.widget, {
 			}
 		);
 	},
+	_setPanelAreaInnerHeight: function( panelHeight ){
+		var self = this;
+		//adjust the scrollable area height only if there are fixed buttons
+		if( self._fixedPanelBtn != null ){
+			var areaHeight = panelHeight - self._fixedPanelBtn.height();
+			self._panelInner.parent().css( 'height',areaHeight );
```
Comment (jasondscott) [inline src/lib/panel.js:353]: What happens on orientation change? Does the hight get updated?

Comment (rdavidwu) [inline src/lib/panel.js:353]: yes it does. It gets called inside _positionPanel() , which gets executed on orientation changes.

**Pull request merged**